src/PrestaShopBundle/Resources/views/Admin/Sell/Order/CreditSlip/Blocks/credit_slip_options.html.twig

```diff
@@ -0,0 +1,56 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain "Admin.Orderscustomers.Feature" %}
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(creditSlipOptionsForm, {
+  'method': 'POST',
+  'action': path('admin_credit_slips_process_options')})
+}}
+<div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">settings</i> {{ 'Credit slip options'|trans({}, 'Admin.Orderscustomers.Feature') }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      {{ ps.form_group_row(creditSlipOptionsForm.options.slip_prefix, {}, {
+        'label': 'Credit slip prefix'|trans({}, 'Admin.Orderscustomers.Feature'),
+        'help': 'Prefix used for credit slips.'|trans({}, 'Admin.Orderscustomers.Help')
+      }) }}
+    </div>
+    {% block credit_slip_options_form_rest %}
+      {{ form_rest(creditSlipOptionsForm) }}
+    {% endblock %}
+  </div>
```

Comment:
Dont hesitate to put some blank lines between big `<div>` structure 😉it eases the reading